recipes/opam/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "opam" %}
+{% set version = "2.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ocaml/opam/releases/download/{{ version }}/opam-full-{{ version }}.tar.gz
+  sha256: de1e3efffd5942e0101ef83dcdead548cc65a93e13986aecb4a264a059457ede
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # C++ definitely needed: https://github.com/ocaml/opam/issues/4452
```

As it can be easy to forget about the `conda_build_config.yaml` file, I might be inclined to suggest drawing attention to it here:
```suggestion
    # Note: Conda-forge's default compiler versions are being overridden in recipe/conda_build_config.yaml
    - {{ compiler('cxx') }} # C++ definitely needed: https://github.com/ocaml/opam/issues/4452
```